packages/test/page-object/src/globals/testHelpers/createDirectLineEmulator.js

```diff
@@ -0,0 +1,127 @@
+import createDeferred from 'p-defer';
```

Comment:
Previously, many of our tests relies on `createDirectLineWithTranscript`.

For newer tests which will send outgoing activities, this `createDirectLineEmulator` should be used instead.